main/src/ca/uwaterloo/flix/language/phase/unification/UnificationCache.scala

```diff
@@ -15,13 +15,15 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import org.sosy_lab.pjbdd.api.DD
 import java.util.concurrent.ConcurrentHashMap
 
 object UnificationCache {
   /**
     * A Global (per-JVM) cache of unification queries.
     */
```

Comment:
Add a comment for each, explain one is for explicit formulas the other for BDDs.